### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by ermik** [graphql-go/relay#37]
Dear @graphql-go peeps! 

It seems this project is dormant and should be clearly labeled so in the README. Please add some information on whether or not this is actively maintained and follows the development of graphql and relay spec.

**Comment by maxbeatty** [comment]
fwiw been working fine for us as-is for last ~6 months

**Comment by koalalorenzo** [comment]
Does anybody know?

**Comment by ermik** [comment]
I have moved over to `gqlgen`.

**Comment by niondir** [comment]
@ermik gqlgen is not working with `github.com/graphql-go/graphql`, is it?

I have a existing project where I would love to generate parts of the schema but not replace my whole implementation.

**Comment by ermik** [comment]
@Niondir no, unfortunately they are not compatible. You may benefit from using your existing schema (by introspecting current `graphql-go/graphql` API) to generate `gqlgen` code; then replace the blank spaces with the logic you had in the original code.

**Comment by everdrone** [comment]
@ermik what about relay connections? I'm trying to figure out how to use `gqlgen` with the Relay spec but it's a bit foggy

**Comment by ermik** [comment]
@everdrone as you saw in https://github.com/99designs/gqlgen/discussions/1900 the connection codegen is not a feature gqlgen supports. However, you can implement (or generate) your own — look at the project referenced in that discussion: https://github.com/MichaelMure/git-bug/tree/master/api/graphql/connections
